keyboards/1upkeyboards/sweet16/keymaps/ridingintraffic/keymap.c

```diff
@@ -0,0 +1,293 @@
+                    process_unicode((0x253B|QK_UNICODE), record);   // Table
+                    process_unicode((0x2501|QK_UNICODE), record);   // Table
+                    process_unicode((0x253B|QK_UNICODE), record);   // Table
+                }
+                return false;
+                break;
+            case TFLIP2:         // ┻━┻︵ \(°□°)/ ︵ ┻━┻
+                if(record->event.pressed){
+                    process_unicode((0x253B|QK_UNICODE), record);   // Table
+                    process_unicode((0x2501|QK_UNICODE), record);   // Table
+                    process_unicode((0x253B|QK_UNICODE), record);   // Table
+                    tap(KC_SPC);
+                    process_unicode((0x0361|QK_UNICODE), record);   // Flippy
+                    tap(KC_SPC);
+                    tap(KC_BSLASH);
+                    register_code(KC_RSFT);
+                    tap(KC_9);
+                    unregister_code(KC_RSFT);
+                    process_unicode((0x00B0|QK_UNICODE), record);   // Eye
+                    process_unicode((0x25A1|QK_UNICODE), record);   // Mouth
+                    process_unicode((0x00B0|QK_UNICODE), record);   // Eye
+                    register_code(KC_RSFT);
+                    tap(KC_0);
+                    unregister_code(KC_RSFT);
+                    tap(KC_SLASH);
+                    tap(KC_SPC);
+                    process_unicode((0x0361|QK_UNICODE), record);   // Flippy
+                    tap(KC_SPC);
+                    process_unicode((0x253B|QK_UNICODE), record);   // Table
+                    process_unicode((0x2501|QK_UNICODE), record);   // Table
+                    process_unicode((0x253B|QK_UNICODE), record);   // Table
+                }
+                return false;
+                break;
+                } 
+    }
+    return true;
+}
+
+
+/* tap dance time */
+void tdexample1(qk_tap_dance_state_t *state, void *user_data) {
+  if (state->count >= 2) {
+    SEND_STRING(EXAMPLESTRING1);
+    reset_tap_dance (state);
+  }
+}
+void tdexample2(qk_tap_dance_state_t *state, void *user_data) {
+  if (state->count >= 2) {
+    SEND_STRING(EXAMPLESTRING2);
+    reset_tap_dance (state);
+  }
+}
+void tdexample3(qk_tap_dance_state_t *state, void *user_data) {
+  if (state->count >= 2) {
+    SEND_STRING(EXAMPLESTRING3);
+    reset_tap_dance (state);
+  }
+}
+void tdexample4(qk_tap_dance_state_t *state, void *user_data) {
+  if (state->count >= 2) {
+    SEND_STRING(EXAMPLESTRING4);
+    reset_tap_dance (state);
+  }
+}
+
+qk_tap_dance_action_t tap_dance_actions[] = {
+    [TD_EXAMPLE1] = ACTION_TAP_DANCE_FN(tdexample1),
+    [TD_EXAMPLE2] = ACTION_TAP_DANCE_FN(tdexample2),
+    [TD_EXAMPLE3] = ACTION_TAP_DANCE_FN(tdexample3),
+    [TD_EXAMPLE4] = ACTION_TAP_DANCE_FN(tdexample4)
+};
+
+// Runs just one time when the keyboard initializes.
+void matrix_init_user(void) {
+tap(RGB_M_P);
+};
+
+// Runs constantly in the background, in a loop.
+void matrix_scan_user(void) {
```

Comment:
If this is the RGB underglow, this is a VERY bad idea. 

You should do this instead: 
https://docs.qmk.fm/#/custom_quantum_functions?id=layer-change-code